pkg/specs/volumes.go

```diff
@@ -75,9 +77,68 @@ func createPostgresVolumes(cluster apiv1.Cluster, podName string) []corev1.Volum
 		)
 	}
 
+	if cluster.ShouldInitDBRunPostInitApplicationSQLRefs() {
+		result = append(result, createPostgresVolumesForPostInitApplicationSQLRefs(
+			cluster.Spec.Bootstrap.InitDB.PostInitApplicationSQLRefs,
+		)...)
+	}
+
 	return result
 }
 
+func createPostgresVolumesForPostInitApplicationSQLRefs(refs *apiv1.PostInitApplicationSQLRefs) []corev1.Volume {
+	length := len(refs.ConfigMapRefs) + len(refs.SecretRefs)
+	res := make([]corev1.Volume, 0, length)
```

Comment:
magic numbers, the user needs to provide the volume for this, if the user wants and will know the size of the volume, other wise, we can be creating a volume that the user will know nothing about

Reply:
user doesn't provider volume, they provider refs to secrets or configmaps only, and we attach volumes to init job spec by the controller.

i assume user cannot change the spec of init job directly

Reply:
also i want to keep the magic number here to make sure the order of implementation to sql files

Reply:
nop, it's not acceptable to keep magic numbers, in that case a directory inside $PGDATA it's a better solution